packages/serverless-orchestration/src/ServerlessHub.js

```diff
@@ -57,9 +57,25 @@ const Web3 = require("web3");
 const { delay, createNewLogger } = require("@uma/financial-templates-lib");
 let customLogger;
 let spokeUrl;
+let spokeUrlTable = {};
```

Comment:
please add a comment in the block comment at the top of this file showing the expected structure of this new parameter. this could go around line 12.